t/.gitignore

```diff
@@ -0,0 +1,8 @@
+data/tests/product/
+fork-share.txt
+raid/hd0
+serial0
+tesseract_opencl_profile_devices.dat
```

Comment:
also this one

Reply:
This is probably from `t/02-test_ocr.t`.